### PR TITLE
tree: swap task name and job order

### PR DESCRIPTION
**Pull request by oliver-sanders**
completes point one of #333 

Swap the job and task name for readability.

Screenshot from 2021-03-04 13-16-53

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

File: src/components/cylc/tree/TreeItem.vue [hunk 1]
```diff
@@ -64,15 +64,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
-          <span class="mx-1">{{ node.node.name }}</span>
           <div v-if="!isExpanded" class="node-summary">
             <!-- Task summary -->
             <job
               v-for="(job, index) in node.children"
               v-cylc-object="job.id"
               :key="`${job.id}-summary-${index}`"
-              :status="job.node.state" />
+              :status="job.node.state"
+              style="margin-left: 0.25em;"
```
Comment (kinow) [inline src/components/cylc/tree/TreeItem.vue:74]: We are using the Vuetify margin-right (`mr`) and margin-left (`ml`) spaces elsewhere. Would that work here too, for consistency? e.g. `ml-1` (`4px`), `ml-2` (`8px`).

Comment (oliver-sanders) [reply src/components/cylc/tree/TreeItem.vue:74]: If we use px as a measurement then this margin won't scale with the font size when it is changed from the user dashboard.

Really surprised vuetify doesn't use relative units like em's rem's, vw, etc, it's pretty essential for responsive and accessible design.

Comment (kinow) [reply src/components/cylc/tree/TreeItem.vue:74]: They have a whole docs section on accessibility, but I haven't checked if these px values vary for other viewports.

So far I think the base layout and most components (including the tree component) have been using vuetify classes.

If you prefer this em value maybe it'd be worth adding a css class + scss var in our styles? Up to you 👍

**Review by kinow — APPROVED** [review]
Much better! Easier to scan the tree

**Review by kinow — APPROVED** [review]

**Pull request merged**